indexer-opts/meta-table.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE meta (
+    indexer_id                  VARCHAR(50)     PRIMARY KEY,
+    last_processed_block_height numeric(20, 0)  NOT NULL,
+    start_block_height          numeric(20, 0)  NOT NULL
```

Comment:
We don't need this file
Moreover, it's outdated

Reply:
Sorry it's an artifact committed by mistake